### PR TITLE
Update host vehicle data with kinematics data

### PR DESCRIPTION
**Pull request by MaikolDrechsler**
Include velocity, acceleration and orientation rate data related to the vehicle coordinate system in the host_vehicle_data message

Signed-off-by: Maikol Drechsler <maikol.drechsler@carissma.eu>

﻿#### Reference to a related issue in the repository
Due to the deprecation of localization and localization_rmse attributes of host_vehicle_data the velocity, acceleration and yaw_rate of the vehicle cannot be described.
The issue had been discussed per e-mail with @ThomasNaderBMW

#### Add a description
Include a new attribute vehicle_kinametics which include the ego velocity, orientation rate and acceleration in the host_vehicle_data. 

**Some questions to ask**:
What is this change?
Include a new attribute, that enables the exchange of vehicle kinematics data.  
What does it fix?
The data lost by the deprecation of the attributes localization and localization_rmse.
Is this a bug fix or a feature? Does it break any existing functionality or force me to update to a new version?
It is a new optional attribute, no update is forced. 
How has it been tested?
It has been tested locally with a C++ implementation. 

#### Take this checklist as orientation for yourself, if this PR is ready for the Change Control Board:
- [X] My suggestion follows the [style and contributors guidelines](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/start_contributing.html).
- [X] I have taken care about the [message documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/commenting_messages.html) and the [fields and enums documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/commenting_fields_enums.html).
- [X] I have done the [DCO signoff](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html#developer-certification-of-origin-dco).
- [X] My changes generate no errors when passing CI tests. 
- [X] I have successfully implemented and tested my fix/feature locally.
- [X] Appropriate reviewer(s) are assigned.

If you can’t check all of them, please explain why.
If all boxes are checked or commented and you have achieved at least one positive review, you can assign the label ReadyForCCBReview!

### CODE REVIEW

File: osi_hostvehicledata.proto [hunk 1]
```diff
@@ -337,6 +342,31 @@ message HostVehicleData
         optional GeodeticPosition geodetic_position = 3;
     }
 
+    //
+    //\brief Current calculated and estimated acceleration, velocity and orientation rate on Inertial Measurement Unit measurements and related sensors.
+    //
+    //This message contains the most accurate information the vehicle knows about its velocity,
```
Comment (ThomasNaderBMW) [inline osi_hostvehicledata.proto:348]: // This
Space is missing

**Review by ThomasNaderBMW — APPROVED** [review]
Maybe correct the missing space, but in general ok for me. Thanks!

File: osi_hostvehicledata.proto (hunk 1 above)
Comment (MaikolDrechsler) [reply osi_hostvehicledata.proto]: Thank you for the review. 
Spaces have been included.

**Comment by pmai** [comment]
CCB 2023-01-16: To be changed as discussed (include references to bbcenter_to_rear in description of reference point for VehicleMotion, reorder fields to match derivative structure, and add comment to VehicleLocalization to indicate middle of bounding box reference point), then rediscussed for merge in next CCB.

**Pull request merged**